### ISSUE TITLE
Loading simplereactroles fail, cannot load dependency

### ISSUE THREAD

**Issue by phantium** [Seputaes/SepRedCogs#1]
Hi,
I'd love to use your simplereactroles but it's not working.

Loading simplereactroles
```
Oct 12 07:10:18 maplebot redbot[12431]: [12/10/2018 07:10] ERROR core_commands _load 102: Package loading failed
Oct 12 07:10:18 maplebot redbot[12431]: Traceback (most recent call last):
Oct 12 07:10:18 maplebot redbot[12431]:   File "/usr/local/lib/python3.6/dist-packages/redbot/core/core_commands.py", line 100, in _load
Oct 12 07:10:18 maplebot redbot[12431]:     await bot.load_extension(spec)
Oct 12 07:10:18 maplebot redbot[12431]:   File "/usr/local/lib/python3.6/dist-packages/redbot/core/bot.py", line 222, in load_extension
Oct 12 07:10:18 maplebot redbot[12431]:     lib = spec.loader.load_module()
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap_external>", line 399, in _check_name_wrapper
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap_external>", line 823, in load_module
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap_external>", line 682, in load_module
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap>", line 265, in _load_module_shim
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap>", line 684, in _load
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap_external>", line 678, in exec_module
Oct 12 07:10:18 maplebot redbot[12431]:   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
Oct 12 07:10:18 maplebot redbot[12431]:   File "/home/bot/cogs/simplereactroles/__init__.py", line 2, in <module>
Oct 12 07:10:18 maplebot redbot[12431]:     from .simplereactroles import SimpleReactRoles
Oct 12 07:10:18 maplebot redbot[12431]:   File "/home/bot/cogs/simplereactroles/simplereactroles.py", line 13, in <module>
Oct 12 07:10:18 maplebot redbot[12431]:     from cog_shared.seplib.classes.basesepcog import BaseSepCog
Oct 12 07:10:18 maplebot redbot[12431]: ModuleNotFoundError: No module named 'cog_shared.seplib'
Oct 12 07:11:24 maplebot redbot[12431]: [12/10/2018 07:11] ERROR core_commands _load 102: Package loading failed
```

Loading required seplib fails too.
```
Oct 12 07:11:24 maplebot redbot[12431]: Traceback (most recent call last):
Oct 12 07:11:24 maplebot redbot[12431]:   File "/usr/local/lib/python3.6/dist-packages/redbot/core/core_commands.py", line 100, in _load
Oct 12 07:11:24 maplebot redbot[12431]:     await bot.load_extension(spec)
Oct 12 07:11:24 maplebot redbot[12431]:   File "/usr/local/lib/python3.6/dist-packages/redbot/core/bot.py", line 230, in load_extension
Oct 12 07:11:24 maplebot redbot[12431]:     lib.setup(self)
Oct 12 07:11:24 maplebot redbot[12431]:   File "/home/bot/cogs/seplib/__init__.py", line 7, in setup
Oct 12 07:11:24 maplebot redbot[12431]:     bot.add_cog(SepLib(bot))
Oct 12 07:11:24 maplebot redbot[12431]:   File "/usr/local/lib/python3.6/dist-packages/redbot/core/bot.py", line 387, in add_cog
Oct 12 07:11:24 maplebot redbot[12431]:     f"The {cog.__class__.__name__} cog in the {cog.__module__} package does "
Oct 12 07:11:24 maplebot redbot[12431]: RuntimeError: The SepLib cog in the seplib.seplib package does not inherit from the commands.Cog base class. The cog author must update the cog to adhere to this requirement.
```

Any chance you could help?

Red Bot Version: 3.0.0rc1.post1

Thanks!

**Comment by Seputaes** [comment]
Hey @phantium ! It looks like the Cog base class is a new requirement in Red V3 that was recently added. I'm seeing several issues being opened for the same issue on other GitHub repos within the last week. The [docs](https://red-discordbot.readthedocs.io/en/v3-develop/guide_cog_creation.html) for V3 haven't even been updated yet to show this is required.  Ah the perils of using pre-release code. 💯 

I'm out of town this weekend for vacation, but I'll look to get this fixed early next week.

**Comment by Seputaes** [comment]
More info on the commit that caused this:

https://github.com/Cog-Creators/Red-DiscordBot/commit/08704032993602a1f38aec3b25804d58a5bb72c6

**Comment by Seputaes** [comment]
Sorry for the delay. This should be fixed as of commit 6f0aa2d1aadeb65531aff26497a16b4b9842bc21. For me, GitHub's UI hasn't updated to show the changes in the master branch, but they're committed to the repo and should be available via a `cog update` command in Red.

Tested on Red commit: Cog-Creators/Red-DiscordBot@8b4e12da8119f65add9ca0b552326612861def51
